jabref-template/listrefs.begin.layout

```diff
@@ -406,27 +406,44 @@ p.infolinks { margin: 0.3em 0em 0em 0em; padding: 0px; }
 <body>
     <!--#include file="header.include" -->
 
+    <div class="bs-header">
+      <div class="container">
+        <br />
```

Comment:
My browser interprets this as `<br>`. Is `<br />` somehow different? I have not seen this before.

Reply:
They're treated equally. `<br />` is equivalent to `<br></br>` and is the common XML form of a self-closing tag. XML wants every tag to be closed, so a `</br>` is necessary. But nothing useful can be between `<br>` and `</br>`, so `<br />` is the commonly used abbreviation.